StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+PaymentMethodAvailability.swift

```diff
@@ -43,7 +43,7 @@ extension PaymentSheet {
     ]
 
     /// A list of `STPPaymentMethodType` that can be saved in PaymentSheet
-    static let supportedSavedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit]
```

Comment:
Nit: I think it'd be more clear to only add link to supportedSavedPaymentMethods, rather than always adding it, then filtering it out for ephemeral keys.

Reply:
Looked into changing this, but it becomes a little messy since `supportedSavedPaymentMethods` is used in lots of places.

This honestly feels more natural to me: Pulling payment methods from `/payment_methods` is the only place where Link isn't supported as a saved payment method.